src/transformers/models/bert_japanese/tokenization_bert_japanese.py

```diff
@@ -132,7 +132,7 @@ def __init__(
         if not os.path.isfile(vocab_file):
             raise ValueError(
                 f"Can't find a vocabulary file at path '{vocab_file}'. To load the vocabulary from a Google pretrained "
-                "model use `tokenizer = BertTokenizer.from_pretrained(PRETRAINED_MODEL_NAME)`"
+                "model use `tokenizer = AutoTokenizer.from_pretrained(PRETRAINED_MODEL_NAME)`"
```

Comment:
It seems to me that the other tokenizers in the `transformers` library specify the specific class of tokenizer here instead of the generic `AutoTokenizer`. Was there any particular reason to prefer `AutoTokenizer` to `BertJapaneseTokenizer`? :slightly_smiling_face: 
```suggestion
                "model use `tokenizer = BertJapaneseTokenizer.from_pretrained(PRETRAINED_MODEL_NAME)`"
```

Reply:
I think it's better to encourage users to use the `AutoTokenizer` class.

Reply:
Thanks a lot for your feedback @sgugger ! In that case, @europeanplaice your proposal is great - you can ignore my previous comment.

@sgugger, Should we take this opportunity to make the same change with other tokenizers that log the same type of message (cf PR #12745)?

Reply:
Yes, that was a great idea!